test/test-adding_image.test

```diff
@@ -0,0 +1,33 @@
+<launch>
+  <arg name="gui" default="false" />
+  <node name="play_face_bag" pkg="rosbag" type="play" args="-l $(find opencv_apps)/test/face_detector_withface_test_diamondback.bag" />
```

Comment:
I'm very sorry but I have merged
https://github.com/ros-perception/opencv_apps/pull/20
so please split .test file to .test file and .launch file. Also add documentation to all args
https://github.com/ros-perception/opencv_apps/blob/indigo/launch/hough_lines.launch#L9-L14